CorePayments/src/main/java/com/paypal/android/corepayments/model/PatchCcoWithAppSwitchEligibilityRequest.kt

```diff
@@ -0,0 +1,68 @@
+package com.paypal.android.corepayments.model
+
+import android.content.Context
+import androidx.annotation.RawRes
+import com.paypal.android.corepayments.LoadRawResourceResult
+import com.paypal.android.corepayments.R
+import com.paypal.android.corepayments.ResourceLoader
+import com.paypal.android.corepayments.model.PatchCcoWithAppSwitchEligibilityRequest.Companion.INTEGRATION_ARTIFACT
+import com.paypal.android.corepayments.model.PatchCcoWithAppSwitchEligibilityRequest.Companion.INTEGRATION_CHANNEL
+import org.json.JSONObject
+
+internal data class PatchCcoWithAppSwitchEligibilityRequest(
+    val variables: Variables
+) {
+    suspend fun create(
+        context: Context,
+        resourceLoader: ResourceLoader = ResourceLoader()
+    ): JSONObject? {
+        @RawRes val resId = R.raw.graphql_query_patch_cco_app_switch_eligibility
+        val query = when (val result = resourceLoader.loadRawResource(context, resId)) {
+            is LoadRawResourceResult.Success -> result.value
+            is LoadRawResourceResult.Failure -> return null
+        }
+        val variablesJson = createVariablesJson(variables)
+        return JSONObject()
+            .put("query", query)
+            .put("variables", variablesJson)
+    }
+
+    private fun createVariablesJson(variables: Variables): JSONObject {
+        val experimentationContext = JSONObject()
+            .put("integrationChannel", variables.experimentationContext.integrationChannel)
+
+        return JSONObject()
+            .put("experimentationContext", experimentationContext)
+            .put(
+                "integrationArtifact",
+                INTEGRATION_ARTIFACT
+            )
```

Comment:
```suggestion
            .put("integrationArtifact", INTEGRATION_ARTIFACT)
```
nit: to help with readability.